visual-recognition/v3.ts

```diff
@@ -111,59 +119,46 @@ class VisualRecognitionV3 extends BaseService {
    * @param {string} [params.acceptLanguage] - The desired language of parts of the response. See the response for
    * details.
    * @param {OutgoingHttpHeaders} [params.headers] - Custom request headers
-   * @param {Function} [callback] - The callback that handles the response
    * @returns {Promise<VisualRecognitionV3.Response<VisualRecognitionV3.ClassifiedImages>>}
    */
-  public classify(params?: VisualRecognitionV3.ClassifyParams, callback?: VisualRecognitionV3.Callback<VisualRecognitionV3.ClassifiedImages>): Promise<VisualRecognitionV3.Response<VisualRecognitionV3.ClassifiedImages>> {
-    const _params = (typeof params === 'function' && !callback) ? {} : extend({}, params);
-    const _callback = (typeof params === 'function' && !callback) ? params : callback;
-
-    return new Promise((resolve, reject) => {
-      const formData = {
-        'images_file': {
-          data: _params.imagesFile,
-          filename: _params.imagesFilename,
-          contentType: _params.imagesFileContentType
-        },
-        'url': _params.url,
-        'threshold': _params.threshold,
-        'owners': Array.isArray(_params.owners) ? _params.owners.join(',') : _params.owners,
-        'classifier_ids': Array.isArray(_params.classifierIds) ? _params.classifierIds.join(',') : _params.classifierIds
```

Comment:
@mediumTaj has this finally been fixed in the API definition or do we still need manual support for Array types here?

Reply:
Actually Mike just said that he updated the code in the service to accept either an array or a comma-separated string. In that case, we don't need this change anymore. Is that true, Taj?

Reply:
Nope, we still need this as comma separated strings. Mike updated in visrec v4 but they didn't think it was worth it to update v3.

Reply:
Thanks for the updates Taj! So Angelo, you'll need to retain this code for the major release.